tests/test_dvc.py

```diff
@@ -321,3 +321,27 @@ def test_no_scm_repo(tmp_dir, mocker):
 
         live = Live(save_dvc_exp=True)
         assert live._save_dvc_exp is False
+
+
+@pytest.mark.parametrize("dvcyaml", [True, False])
+def test_warn_on_dvcyaml_output_overlap(tmp_dir, mocker, dvcyaml):
```

Comment:
You should be able to use `mocked_dvc_repo` and update only the properties needed (see examples above int his file)